api/reporting/schemas/input/excluded_columns/address.txt

```diff
@@ -0,0 +1,2 @@
+id
+user_id
```

Comment:
How do you connect an address to a user without this?

More generally, how did you choose which columns to exclude?

Reply:
I'm manually putting it in the `JOIN` statement We only use the `DbAddress` class in `UserDao`.

The schema doesn't include columns that are implementation details and don't have user-facing or admin-facing reality.